src/components/chat/ChatInput.vue

```diff
@@ -84,12 +84,11 @@
         :placeholder="$t('chatInput.placeHolder')"
       />
       <q-space />
-      <q-btn
-        dense
-        flat
-        icon="send"
-        @click="sendMessage"
-      />
+      <div @mousedown.prevent="sendMessage">
```

Comment:
It seems like it would be more appropriate to use:

```
          <q-btn @click.capture.stop="sendMessage" icon="more vert"></q-btn>
```

On q-btn.